Fixes this error:

pplite_poly.cc: In destructor ‘pplite::apron::pplite_internal::~pplite_internal()’:
pplite_poly.cc:46:7: error: ‘flint_cleanup’ was not declared in this scope
   46 |       flint_cleanup();
      |       ^~~~~~~~~~~~~

--- pplite/pplite_poly.cc.orig	2024-06-07 09:36:08.000000000 -0600
+++ pplite/pplite_poly.cc	2024-06-13 08:21:55.536500683 -0600
@@ -18,6 +18,7 @@
 #include "ap_pplite.h"
 #include "pplite_user.hh"
 #include "pplite_poly.hh"
+#include <flint/flint.h>
 
 #include <cassert>
 
